restdocs-api-spec-jsonschema/src/test/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGeneratorTest.kt

```diff
@@ -289,6 +292,89 @@ class JsonSchemaFromFieldDescriptorsGeneratorTest {
         thenSchemaIsValid()
     }
 
+    @Test
+    fun should_specify_generic_items_type_in_array_when_descriptor_doesnt_contain_itemsType_in_additionalParameters() {
```

Comment:
The "arrange" and "act" of this test is the same as the existing test `should_generate_schema_for_top_level_array_of_any`.

I suggest to extend the existing test with the additional assertions from this test and remove this test.